packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1036,6 +1049,13 @@ export class GarbageCollector implements IGarbageCollector {
         return gcStats;
     }
 
+    private sweepNodes(sweepReadyNodes: string[]) {
```

Comment:
nit: Maybe move this below just before the definition of removeDeletedNodesFromGCState so that the sweep methods are together.